### ISSUE TITLE
Update Lithology

### ISSUE THREAD

**Issue by nickmachairas** [DIGGSml/diggs-schema#10]
We are seeking comments/recommendations on updating the lithology codelist as shown below. This post is also kicking off the [Codelist Updates](https://github.com/DIGGSml/diggs-schema/projects/1) project page where we will be keeping track of the development process.

`diggs:Lithology/diggs:classificationCode`

Code | Name | Description | Data type | UOMType | Authority
:--: | -- | -- | -- | -- | :--:
SP-SM | Poorly-graded sand with silt | Poorly-graded sand with silt |   |   | USCS
GW-GC | Well-graded gravel with clay (or silty clay) | Well-graded gravel with clay (or silty clay) |   |   | USCS
CL | Lean clay | Lean clay |   |   | USCS
GP-GM | Poorly-graded gravel with silt | Poorly-graded gravel with silt |   |   | USCS
CH | Fat clay | Fat clay |   |   | USCS
GC | Clayey gravel | Clayey gravel |   |   | USCS
GM | Silty gravel | Silty gravel |   |   | USCS
GP-GC | Poorly-graded gravel with clay (or silty clay) | Poorly-graded gravel with clay (or silty clay) |   |   | USCS
CL-ML | Silty clay | Silty clay |   |   | USCS
SP-SC | Poorly-graded sand with clay | Poorly-graded sand with clay |   |   | USCS
GW-GM | Well-graded gravel with silt | Well-graded gravel with silt |   |   | USCS
GP | Poorly-graded gravel | Poorly-graded gravel |   |   | USCS
SW-SM | Well-graded sand with silt | Well-graded sand with silt |   |   | USCS
SW | Well-graded sand | Well-graded sand |   |   | USCS
GC-GM | Silty, clayey gravel | Silty, clayey gravel |   |   | USCS
GW | Well-graded gravel | Well-graded gravel |   |   | USCS
ML | Silt | Silt |   |   | USCS
OH | Organic clay | Organic clay |   |   | USCS
SC-SM | Silty, clayey sand | Silty, clayey sand |   |   | USCS
MH | Elastic silt | Elastic silt |   |   | USCS
SC | Clayey sand | Clayey sand |   |   | USCS
OL | Organic clay | Organic clay |   |   | USCS
SM | Silty sand | Silty sand |   |   | USCS
SW-SC | Well-graded sand with clay | Well-graded sand with clay |   |   | USCS
SP | Poorly-graded sand | Poorly-graded sand |   |   | USCS
LFS | Loamy fine sand | Loamy fine sand |   |   | USDA
SIC | Silty clay | Silty clay |   |   | USDA
COS | Coarse sand | Coarse sand |   |   | USDA
LFS | Loam | Loam |   |   | USDA
LCOS | Loamy coarse sand | Loamy coarse sand |   |   | USDA
SC_ | Sandy clay | Sandy clay |   |   | USDA
CL_ | Clay loam | Clay loam |   |   | USDA
SL | Sandy loam | Sandy loam |   |   | USDA
COSL | Coarse sandy loam | Coarse sandy loam |   |   | USDA
S | Sand | Sand |   |   | USDA
VFS | Very fine sand | Very fine sand |   |   | USDA
FS | Fine sand | Fine sand |   |   | USDA
LS | Loamy sand | Loamy sand |   |   | USDA
SICL | Silty clay loam | Silty clay loam |   |   | USDA
SCL | Sandy clay loam | Sandy clay loam |   |   | USDA
FSL | Fine sandy loam | Fine sandy loam |   |   | USDA
VFSL | Very fine sandy loam | Very fine sandy loam |   |   | USDA
SIL | Silt loam | Silt loam |   |   | USDA
LVFS | Loamy very fine sand | Loamy very fine sand |   |   | USDA
SI | Silt | Silt |   |   | USDA
A-1-a | Gravel and/or stone fragments | Gravel and/or stone fragments |   |   | AASHTO
A-1-b | Gravel and/or stone fragments with sand | Gravel and/or stone fragments with sand |   |   | AASHTO
A-3 | Fine sand | Fine sand |   |   | AASHTO
A-3-a | Coarse and fine sand | Coarse and fine sand |   |   | ODOT
A-2-4 | Gravel and/or stone fragments with sand and silt | Gravel and/or stone fragments with sand and silt |   |   | AASHTO
A-2-5 | Gravel and/or stone fragments with sand and silt | Gravel and/or stone fragments with sand and silt |   |   | AASHTO
A-2-6 | Gravel and/or stone fragments with sand, silt and clay | Gravel and/or stone fragments with sand, silt and clay |   |   | AASHTO
A-2-7 | Gravel and/or stone fragments with sand, silt and clay | Gravel and/or stone fragments with sand, silt and clay |   |   | AASHTO
A-4 | Silty soils | Silty soils |   |   | AASHTO
A-4-a | Sandy silt | Sandy silt |   |   | ODOT
A-4-b | Silt | Silt |   |   | ODOT
A-5 | Elastic silt and clay | Elastic silt and clay |   |   | AASHTO
A-6 | Clayey soils | Clayey soils |   |   | AASHTO
A-6-a | Silt and clay | Silt and clay |   |   | ODOT
A-6-b | Silty clay | Silty clay |   |   | ODOT
A-7-5 | Elastic clay | Elastic clay |   |   | AASHTO
A-7-6 | Clay | Clay |   |   | AASHTO
A-8 | Organic soils | Organic soils |   |   | AASHTO
A-8-a | Organic silt | Organic silt |   |   | ODOT
A-8-b | Organic clay | Organic clay |   |   | ODOT
sod_and_topsoil | Sod and topsoil | Sod and topsoil |   |   | ODOT
pavement_or_base | Pavement or base | Pavement or base |   |   | ODOT
uncontrolled_fill | Uncontrolled fill | Uncontrolled fill |   |   | ODOT
bouldery_zone | Bouldery zone | Bouldery zone |   |   | ODOT
peat | Peat | Peat |   |   | ODOT

**Comment by AC107** [comment]
This looks like a good starting point.  If there are other classification systems or if someone notes a problem with one of these, please provide a comment so that the committee can review this codelist and make appropriate modifications.

**Comment by AC107** [comment]
Should we change the name from Update:lithology to Update Codelist:classificationCode  or just Update:classificationCode if it’s in some sort of group of all codelists?

I think the lithology may be confusing?
May be a lot of codelists related to lithology  (rock type, geoelogy?)


—
Allen Cadden / Principal

**Comment by Rogerchandler** [comment]
The AGS in the UK has a list for lithology here http://www.agsdataformat.com/User/pickv4.php?abbr=GEOL_LEG (requires login to website which is free).  I won't add them to the list above I don't think they will "merge together" well as the UK and US have different views on classification systems.   But this may be of interest to others.  If the AGS adopt DIGGS then they will link to their list rather for the UK rather than trying to come up with a list for worldwide usage.

**Comment by AC107** [comment]
Roger, you make a good point here.  Any user or entity can link their own lists and for AGS this may be the best approach.  We should focus on this being a good general list for those that do not have their own definitions already defined and those that are already using systems similar or common to those used in the US as reflected here.

**Comment by sdeaton75** [comment]
I believe our Authority needs to be more specific.  USCS could be field classification vs lab classification.  They are fundamentally similar but different systems.  They have all of the same USCS symbols except the lab classification has a CL-ML classification. So, we could duplicate all the codes except CL-ML and then have 2 different Authorities, ASTM D2487 and ASTM D2488.

**Comment by AC107** [comment]
As we were discussing on the DIGGS call this am... should the "Code" require an "Authority"?  issues such as USCS CL (Lean Clay) and USDA CL_ (Clayey Loam).  If we required the Authority, we would not have to have CL and CL_

**Comment by AC107** [comment]
Will it be confusing or incorrect if you have to jump between ODOT and AASHTO fro various classifications?  A-6..  A-6a and A-6b .  (I should also do some research on how they tell between silt and clay and silty clay)   Is authority a required field?

**Comment by sdeaton75** [comment]
Yes - I believe it is confusing and potentially misleading.  It causes problems for users, software developers, etc.  If all 50 state DOTs decide to disregard the standards then there is little point in having the standard in the first place other than for commercial work, which then means you still have the tail wagging the dog (e.g. DOT's dictating a different way of doing things similar to the problem you have with DOT's mandating gINT project files must be provided which requires consultants to change their process for each DOT).  

There is a reason why there are ASTM standards and AASHTO standards,  Individual states doing their own thing is problematic and lessens the value of the data, and potentially provides misleading data in scenarios where people do not know that ODOT's classification system is different than MODOT's classification which is different than LADOT, etc.  This also means that doing any data mining is unreliable because the differences in classification using these systems.
